filters/utilization_threshold.cpp

```diff
@@ -97,6 +97,7 @@ Try<Nothing> UtilizationThresholdFilter::consume(const ResourceUsage& product) {
     // Send only when node utilization is not too high.
     if ((totalCpuUsage / totalSlaveCpus.get()) < this->utilizationThreshold) {
       // Continue pipeline.
+      SERENITY_LOG(INFO) << "Producing.";
```

Comment:
For the filters debug it’s enough and really helpful. For production we will remove it or do it as a vlog (: